app/services/plans/override_service.rb

```diff
@@ -29,7 +29,7 @@ def call
           return taxes_result unless taxes_result.success?
         end
 
-        plan.charges.each do |charge|
+        plan.charges.includes(:group_properties).each do |charge|
```

Comment:
Rails/FindEach: Use `find_each` instead of `each`. (https://rails.rubystyle.guide#find-each)